src/System.Net.NameResolution/tests/FunctionalTests/GetHostByNameTest.cs

```diff
@@ -106,14 +106,14 @@ public void DnsObsoleteGetHostByName_IPv6String_ReturnsOnlyGivenIP()
         public void DnsObsoleteGetHostByName_EmptyString_ReturnsHostName()
         {
             IPHostEntry entry = Dns.GetHostByName("");
-            Assert.Contains(Dns.GetHostName(), entry.HostName);
+            Assert.Contains(Dns.GetHostName(), entry.HostName, StringComparison.OrdinalIgnoreCase);
```

Comment:
It should be case insensitive for just a-z, based on that RFC. @tarekgh is there a built in way to be ASCII case insensitive, sensitive otherwise? If not you can trivially do it (in almost one line) of course.

Reply:
Looks like there isn't a built in way, on ASCIIEncoding.